src/assets/cta.scss

```diff
@@ -67,7 +72,12 @@ a {
     /* Zep */
     font-size: 13px;
     line-height: 1.46;
-    color: var(--primary-blue);
+    color: var(--pastel-blue);
+  }
+
+  &.ra21-access-text:focus {
+    outline: none;
+    text-decoration: underline;
   }
```

Comment:
An outline during "focus" state looked weird in the link below the button, the underline is more natural.